boilerplate/addict.css

```diff
@@ -0,0 +1,247 @@
+}
+.addict.label {
+  position: absolute;
+  height: 1.7rem;
+  line-height: 2rem;
+  bottom: 0;
+  color: #999;
+  transition: all .3s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+}
+.addict.input {
+  width: 100%;
+  font-size: 100%;
+  border: 0;
+  padding: 0;
+  background-color: transparent;
+  height: 2rem;
+  line-height: 2rem;
+  border-bottom: 1px solid #eee;
+  color: #777;
+  transition: all .2s ease-in;
+}
+.addict.input:focus {
+  outline: 0;
+  border-color: #ccc;
+}
+
+/* Using required and a faux pattern to see if input has text from http://stackoverflow.com/questions/16952526/detect-if-an-input-has-text-in-it-using-css */
+.addict.input:focus + .label,
+.addict input:valid + .label {
+  transform: translateY(-100%);
+  font-size: 0.75rem;
+  color: #ccc;
+}
+
+/* Button */
+.addict.btn {
+  cursor: pointer;
+  background-color: transparent;
+  border: 0;
+  font-size: 0.75rem;
+  height: 2.5rem;
+  line-height: 2.5rem;
+  padding: 0 1.5rem;
+  color: white;
+  background: #999;
+  text-transform: uppercase;
+  border-radius: .25rem;
+  letter-spacing: .2em;
+  transition: background .2s;
+}
+.addict.btn:focus {
+  outline: 0;
+}
+.addict.btn:hover,
+.addict.btn:focus {
+  background: #000;
+}
+
+.addict a {
+  text-decoration: none;
+  color: #777;
+  font-size: 80%;
+}
+
+.addict a:hover {
+  color: #000;
+  transition: color .2s;
+}
+
+.addict.error-message {
+  margin-bottom: 0;
+  margin-top: 0;
+  color: #F00;
+  font-size: 80%;
+}
+
+/* Intro animation */
+@keyframes intro {
+  from {
+    opacity: 0;
+    top: 0;
+  }
+  to {
+    opacity: 1;
+    top: 50%;
+  }
+}
+/*body {
+  position: relative;
+  margin: 0;
+  font-family: 'Work Sans', Arial, Helvetica, sans-serif;
+  min-height: 100%;
+  background: linear-gradient(to bottom, #68EACC 0%, #497BE8 100%);
+  color: #777;
+}
+
+.addict.container {
+    position: absolute;
+    width: 24em;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%,-50%);
+}
+
+.addict.flex {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between; }
+
+.addict.field-whole {
+  flex-basis: 100%; }
+
+.addict.field-half {
+  flex-basis: 48.75%; }
+
+.addict.form-container {
+    position: relative;
+    background: #DEDEDE;
+    padding: 3em;
+    border-radius: .25em;
+    -webkit-filter: drop-shadow(0 0 2em rgba(0,0,0,0.2));
+    transition: opacity .4s ease-in-out, height .4s ease-in-out, transform .4s cubic-bezier(0.175, 0.885, 0.32, 1.275), padding .4s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+}
+
+.addict input[type="text"],
+.addict input[type="password"] {
+  width: 16em;
+  padding: 1em;
+  border: none;
+  border-bottom: solid 3px #c9c9c9;
+  transition: border 0.3s;
+}
+
+.addict input[type="text"]:focus,
+.addict input[type="password"]:focus,
+.addict input[type="password"].focus,
+.addict input[type="text"].focus {
+  outline: transparent;
+  border-bottom: solid 3px #969696;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+*/
```

Comment:
I missed rule number two of Pull Requests: always review what you're submitting before actually hitting the "Create PR" button :disappointed: